tests/target/configs/indent_style/block_tab_spaces_call.rs

```diff
@@ -4,7 +4,7 @@
 
 // #1427
 fn main() {
-  exceptaions::config(move || {
+  exceptions::config(move || {
```

Comment:
https://github.com/rust-lang/rustfmt/issues/1427 spelled it like this, I'm not sure why it changed...